### ISSUE TITLE
Adding ncclimo to diags

### ISSUE THREAD

**Issue by zshaheen** [zshaheen/acme_diags#81]
A long time ago, some people mentioned that we should add ncclimo into our diagnostics, so users don't have to do that preprocessing beforehand. However, this is already handled by the ACME Workbench. What are your thoughts @chengzhuzhang?

**Comment by golaz** [comment]
The workbench generates two types of files: climatologies and time series. Both are created by ncclimo. Currently, ACME Diagnostics can only use climatologies as input files. As an additional feature, ACME Diagnostics could support reading time series files and compute climatologies on the fly. This would give us additional flexibility in terms of specifying time intervals (for model and observations). If we go this way, it would make most sense to use CMDS to compute climatologies from time series.

**Comment by chengzhuzhang** [comment]
I strongly agree that we should support read in time series files, so that we can have variability diags. My only concern is if CDMS can handle the large data processing. Seems like most are still using nco over CDMS to generate climatology for ACME? We need to find out if it is practical to use CDMS instead for ACME model output (already regridded).
